### PR TITLE
[AV-290] RASAeroImport class skeleton

### PR DESCRIPTION
**Pull request by AyberkY**
Wrote created the skeleton of a class called `RASAeroImport` that'll handle parsing and looking up aerodynamic coefficients from RASAero data.

### CODE REVIEW

**Comment by AyberkY** [comment]
[[AV-290] Write the skeleton of a RASAeroImport class that imports all the necessary data from RASAero files](https://trello.com/c/5Gyl2xAi/290-av-290-write-the-skeleton-of-a-rasaeroimport-class-that-imports-all-the-necessary-data-from-rasaero-files)

**Comment by gautamdayal** [comment]
lgtm

**Comment by ktt3** [comment]
Does this incorporate interpolation of the data?

**Comment by ktt3** [comment]
The data given by the RASAero fetching script gives out the following variables: CD Power-Off, CD Power-On, CA Power-Off, CA Power-On, CN Total, CP Total. May want to adjust functions to account for these parameters.

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,8 @@ set(CMAKE_CXX_STANDARD 17)
 
 set(LIB ${CMAKE_SOURCE_DIR}/lib)
 add_subdirectory(${LIB}/spdlog)
-
+add_subdirectory(${LIB}/eigen)
+add_subdirectory(${LIB}/rapidcsv)
 
 
 add_executable(ISS_SILSIM src/main.cpp
```
Comment (NPIPHI) [inline CMakeLists.txt:13]: RASAeroImort.cpp needs to be added to the sources in the add_executable call

**Review by NPIPHI — CHANGES_REQUESTED** [review]
CMakeLists needs a little change, other than that LGTM

File: CMakeLists.txt (hunk 1 above)
Comment (AyberkY) [reply CMakeLists.txt:13]: Fixed

**Comment by AyberkY** [comment]
> The data given by the RASAero fetching script gives out the following variables: CD Power-Off, CD Power-On, CA Power-Off, CA Power-On, CN Total, CP Total. May want to adjust functions to account for these parameters.

I've tweaked the getter functions to reflect this. Other than these, there is also a `get_CD_protuberance` function which I think would be nice to have.

**Pull request merged**